rllib/models/utils.py

```diff
@@ -95,41 +98,53 @@ def get_activation_fn(name: Optional[str] = None, framework: str = "tf"):
     if callable(name):
         return name
 
+    name_lower = name.lower() if isinstance(name, str) else name
+
     # Infer the correct activation function from the string specifier.
     if framework == "torch":
-        if name in ["linear", None]:
+        if name_lower in ["linear", None]:
             return None
-        if name == "swish":
-            from ray.rllib.utils.torch_utils import Swish
 
-            return Swish
         _, nn = try_import_torch()
-        if name == "relu":
+        # First try getting the correct activation function from nn directly.
```

Comment:
`torch.nn.SiLU` now does exist, no need to point to our own implementation (removed)